### ISSUE TITLE
Build Failure in Tool-Solutions/docker/pytorch-aarch64

### ISSUE THREAD

**Issue by vishwascm** [ARM-software/Tool-Solutions#254]
I tried to build Pytorch using latest commit in Tool-Solutions/docker/pytorch-aarch64 and found the following error:

Command used 
```Tool-Solutions/docker/pytorch-aarch64/build.sh ```

Error
```
repo_url=https://review.mlplatform.org/ml/ComputeLibrary/
++ echo https://review.mlplatform.org/c/ml/ComputeLibrary/+/12818/1
++ awk -F/ '{print "refs/changes/" substr($(NF-1),length($(NF-1))-1,2) "/" $(NF-1) "/" $(NF)}'
+ refname=refs/changes/18/12818/1
+ git fetch https://review.mlplatform.org/ml/ComputeLibrary/ refs/changes/18/12818/1
From https://review.mlplatform.org/ml/ComputeLibrary
 * branch                  refs/changes/18/12818/1 -> FETCH_HEAD
+ git cherry-pick FETCH_HEAD
Committer identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'root@ip-172-32-5-129.(none)')
```

My git credentials are set properly.

**Comment by jondea** [comment]
Hi @vishwascm, thanks for the bug report!

I'm not sure how your machine is set up, but it does look like you haven't set up an email address and name for `git`. Have you tried running those two suggested commands (even with some example email and name)?

That being said, I think we can fix this another way. Could you try adding `--no-commit` to the `cherry-pick` here please and let me know if it works for you?
https://github.com/ARM-software/Tool-Solutions/blob/4d593cb2559f19da9b92929b1f5f7ea5e1f9b8f2/docker/pytorch-aarch64/get-source.sh#L59

**Comment by vishwascm** [comment]
@jondea, thanks for reply.
I will create a new issue for scaling Issue.

**Comment by jondea** [comment]
Fixed by #257
